.env_sample

```diff
@@ -9,3 +9,4 @@ export EMAIL_USE_TLS=True
 export EMAIL_HOST_USER='your email'
 export EMAIL_HOST_PASSWORD='email pass'
 export EMAIL_PORT=587
+export FRONT_END_URL='http://localhost:8080/login'
```

Comment:
Won't we need to then change this again later to the heroku link??

Reply:
Great question @wasibani-roy, I agree with you, We will have to define this in the environment variables as we change environments,That's why its now in an env file on local.

Reply:
Cool